backend/app/api/routes/chat.py

```
@@ -0,0 +1,126 @@
+            request.model_id = default_model
+        else:
+            raise HTTPException(
+                status_code=400,
+                detail="No available models. Please configure at least one API key."
+            )
+
+    # Get or create session
+    if request.session_id and session_store.session_exists(request.session_id):
+        session = session_store.get_session(request.session_id)
+    else:
+        session = None
```

Ruff (0.14.10)

Local variable `session` is assigned to but never used

Remove assignment to unused variable `session`

(F841)